uvicorn/main.py

```diff
@@ -256,6 +256,9 @@ def print_version(ctx, param, value):
     is_eager=True,
     help="Display the uvicorn version and exit.",
 )
+@click.option(
+    "--app-dir", "app_dir", default=".", help="Specifty application directory."
```

Comment:
I think we can add `show_default=True`, so that `.` shows as a default when running `$ uvicorn --help`:

```suggestion
    "--app-dir", "app_dir", default=".", show_default=True, help="Specifty application directory."
```

Reply:
Should we update help to "Look for `APP` in the specified directory. Defaults to `.`."?